apps/posts/src/views/comments/hooks/use-filter-state.ts

```
@@ -5,53 +5,57 @@ import type {Filter} from '@tryghost/shade';
                 // Create Date objects in user's local timezone, then convert to UTC
                 const startOfDay = new Date(dateValue + 'T00:00:00').toISOString();
                 const endOfDay = new Date(dateValue + 'T23:59:59.999').toISOString();
-                    
+
                 parts.push(`created_at:>='${startOfDay}'+created_at:<='${endOfDay}'`);
             }
             break;
 
-        case 'body': 
+        case 'body':
             const value = filter.values[0] as string;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)